### PR TITLE
Add user API token

### LINKED ISSUE #46: Authenticate when using REST API
**Is your feature request related to a problem? Please describe.**
When attempting to access the JSON of a record (e.g., /dm_items/4371/json) in PostMan, the HTML login page is returned. Accessing that endpoint using a browser that's already logged into the DM Log successfully returns the JSON for that record.

**Describe the solution you'd like**
In addition to the authentication for the GUI, we need a way to access the JSON data using the REST API.

**Additional context**
The DM Log uses the Devise Gem for authentication. See https://stackoverflow.com/questions/39397554/how-do-i-authenticate-using-devise-in-a-rails-rest-api-app

### PR DESCRIPTION
**Pull request by bonniegee**
Fixes #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +99,9 @@ end
 puts "There are now #{DmItem.count} rows in the digital media items table"
 ```
 
+## Usage
+To access JSON data via the API, requests should have the format `/api/dm_items/10?user_email=example@example.com&user_token=1G8_s7P-V-4MGojaKD7a`. The token is available in the app GUI.
```
Comment (helrond) [inline README.md:103]: Is this the only endpoint that's available via the API? If not, you may want to say something like "API requests should include user email and token as URL parameters, for example..."

Comment (bonniegee) [reply README.md:103]: Yes, right now that is the only endpoint available via the API

**Review by p-galligan — APPROVED** [review]
Generally looks good. I agree with Hillel that the JSON data example is a little clunky, but I think it works for now.

**Comment by bonniegee** [comment]
Thanks all! I'm going to address the JSON data example when I tackle issue #34 .

**Pull request merged**